data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/SortingConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class SortingConfiguration {
```

Comment:
I think we need to rework this config structure a bit, given that both the order and key are specified for each sort entry like this

```
"sort": [
    {
      "line_id": {
        "order": "desc"
      },
     {
      "line_id_two": {
        "order": "asc"
      },
    },
]
```

So the config in the yaml can look like this

```
sorting:
  -  sort_key: "first_key"
     order: "asc"
  -  sort_key: "second_key"
     order: "desc"
```

You can see a similar structure here for `entries` config in the add_entries processor (https://github.com/opensearch-project/data-prepper/blob/42cf9d09d71351b38baacb2dd9c8aad387625ea2/data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/AddEntryProcessorConfig.java#L77)